### PR TITLE
Expose tf.lookup.TextFileIndex in 2.0

### LINKED ISSUE #26622: Export tf.lookup.TextFileIndex in 2.0
**System information**
- TensorFlow version: `2.0.0.dev20190311`
- Are you willing to contribute it: No

**Describe the feature and the current behavior/state.**

The exported class `tf.lookup.TextFileInitializer` takes as argument `key_index` and `value_index` but the enum `TextFileIndex` is not available via the API.

The `tf.lookup` module should export this enum.

**Will this change the current api? How?**

This will add a new symbol covered by backward compatibility guarantees.

**Who will benefit with this feature?**

Users of `tf.lookup.TextFileInitializer`.

### PR DESCRIPTION
**Pull request by yongtang**
This fix tries to address the issue raised in #26622 where tf.lookup.TextFileIndex was not exposed from public API though it is used by tf.lookup.TextFileInitializer which actually needs tf.lookup.TextFileIndex.

This fix exposed the tf.lookup.TextFileIndex.

This fix fixes #26622.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: tensorflow/python/ops/lookup_ops.py [hunk 1]
```diff
@@ -419,6 +419,7 @@ def initialize(self, table):
     return init_op
 
 
+@tf_export("lookup.TextFileIndex")
 class TextFileIndex(object):
```
Comment (rohan100jain) [inline tensorflow/python/ops/lookup_ops.py]: Lets add some documentation here describing what these constants mean and how can one use them. I think they are described in the file elsewhere.

**Review by rohan100jain — CHANGES_REQUESTED** [review]

**Comment by yongtang** [comment]
@rohan100jain @alextp Thanks for the review. The doc string for `tf.lookup.TextFileIndex` has been added. Please take a look.

File: tensorflow/python/ops/lookup_ops.py (hunk 1 above)
Comment (alextp) [reply tensorflow/python/ops/lookup_ops.py]: The documentation helps a lot, but I still don't know which endpoints in the TF API accept a TextFileIndex. Mentioning it here is helpful

**Comment by alextp** [comment]
Ok, looks good. I'll approve after the next tf api owners sync.

**Review by alextp — APPROVED** [review]

**Pull request merged**